docs/robots/warthog/maintenance_warthog.mdx

```diff
@@ -14,35 +14,127 @@ import Support from "/components/support.mdx";
+
+Note that the `battery_system_current` topic will be positive when the batteries are charging, indicating that
+current is flowing into the batteries. The lights on the charger unit itself will also indicate when the device is
+charging.
+
+#### To Connect the Charger
+
+1. Plug in the charger DC output cable (heavier gauge cable).
+2. Plug in the charger auxiliary control cable.
+3. Plug the charger into AC power (directly to the wall outlet, no extension cords).
+4. This can be done with or without power on Warthog being on (however the note above applies: at least once every 2-3 weeks, the charge
+   process needs to be allowed to fully balance the system).
+
+#### To Disconnect the Charger
+
+1. Unplug the charger from AC power.
+2. Unplug the charger auxiliary cable.
+3. Unplug the charger DC output cable (heavier gauge cable).
+
+#### Battery Pack Specifications
+
+Key battery system specifications are listed in the table below.
+
+| Specification                     | Measurement                         |
+| :-------------------------------- | :---------------------------------- |
+| System Voltage ( nominal )        | 51.2V ( up to 59 V while charging ) |
+| System Capcity                    | 118 Ah                              |
+| System Energy                     | 6.041 kWh                           |
+| Continuous Discharge Current      | 150 A                               |
+| Peak Discharge Current ( 30 sec ) | 300 A                               |
+| Discharge Temperature Range       | -10 °C to 50 °C                     |
+| Charge Temperature Range          | 0 °C to 45 °C                       |
+
+### General Battery Safety and Maintenance
+
 Always exercise caution and observe the following safety practices when connecting, disconnecting or handling batteries:
 
-- Batteries are high voltage, high current
-- Battery packs must be properly fastened down to ensure they do not move when the Warthog is in operation.
-- Ensure that the battery packs are evenly distributed throughout the Warthog to maximize stability.
+- Batteries are high voltage, high current.
+- Batteries must be properly fastened down to ensure they do not move when the Warthog is in operation.
+- Ensure that the batteries are evenly distributed throughout the Warthog to maximize stability.
 - Battery levels on the unit should be checked on a regular basis.
   It's important to maintain the battery voltage at a suitable level for proper operation.
-- When additional battery packs are added to the system, it's important to connect the positive terminal first to the main power of the Warthog before connecting ground.
-- When installing additional battery packs, disconnect the ground on all battery packs presently in the unit before connecting the positive terminal of the new battery packs.
+- When additional batteries are added to the system, it's important to connect the positive terminal first to the main power of the Warthog before connecting ground.
```

Comment:
I suggest we remove lines related to the User adding more batteries in parallel to the system.

Reply:
Ok, removed